package.json

```diff
@@ -43,10 +44,20 @@
     "pkgstamp": "node ./scripts/pkgstamp.js",
     "pkgbump": "node ./scripts/pkgbump.js && node ./scripts/pkgcheck.js --fix",
     "pkgset": "node ./scripts/pkgset.js",
-    "depcheck": "node ./scripts/depcheck.js"
+    "depcheck": "node ./scripts/depcheck.js",
+    "changelog:unreleased": "lerna-changelog"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/accordproject/web-components"
+  },
+  "changelog": {
+    "labels": {
+      "type: breaking change 💥": ":boom: Breaking Change",
+      "type: enhancement ✨": ":sparkles: Enhancement",
+      "type: bug 🐛": ":bug: Bug",
+      "type: chore 🧼": ":soap: Chore",
+      "type: documentation 📝": ":memo: Documentation"
```

Comment:
So it looks like we should maybe do a more comprehensive mapping of [our labels](https://github.com/accordproject/web-components/labels) to match what is in [our current releases](https://github.com/accordproject/techdocs/wiki/Releases#release-notes-structure) (that related to `web-components`)?

Reply:
I found that in cases where multiple labels have been applied, this tool adds the PR in multiple places. 

Personally, I also find the difference between a "Feature Request", "Enhancement" and "Styling" to be too subtle for a release note.

Also, "Feature Request" specifically doesn't make much sense as a category for a release note.

Reply:
BTW, I also tried mapping "Feature Request" and "Styling" labels to ":sparkles: Enhancement" but that leads to a bug in the tool where we end up with 3 copies of the "Enhancement" section and the "Feature request" and "Styling" PRs are lost.

Reply:
Well I think `Feature Request` would be `New Feature`.

So actually, what happens to merged PRs that are `Type: Feature Request` right now? Are they not included?

Reply:
> So actually, what happens to merged PRs that are `Type: Feature Request 🛍️` right now? Are they not included?

That's right, they get ignored.

What's the difference between `Type: Enhancement ✨ `, `Type: Feature Request 🛍️`  (as a GitHub label ) and `🚀 New Feature` ?

Reply:
`Type: Enhancement ✨` and `Type: Feature Request 🛍️`` are labels for PRs and Issues, `🚀  New Feature` is a tag in the release note. More PRs are tagged `Type: Feature Request 🛍️`` than `Type: Enhancement ✨`, so we don't want to be ignoring those.

Reply:
Let me rephrase...

In a release note, personally, I wouldn't expect a category called "Feature Request". By the time that it gets to that stage, the change is a feature (or an enhancement). I agree that "Feature Request" makes sense as a label for issues, but less so for PRs.

In that context, we shouldn't retain both and map them to the same `🚀 New Feature` section in a release note because of the bug that I mentioned in `lerna-changelog`. We can only have unique targets in the mapping.

So my proposal is to not use the label `Type: Feature Request 🛍️` for PRs and instead use `Type: Enhancement ✨`. An alternative would be to also label every PR that has `Type: Feature Request 🛍️` with `Type: Enhancement ✨` (because the former would be ignored)

Reply:
Okay yeah this makes sense. So Issues => `Type: Feature Request 🛍️`, PRs => `Type: Enhancement ✨`.

Could you as a next step update https://github.com/accordproject/web-components/wiki to have the `Release Workflow` Guide to not point to [`techdocs`](https://github.com/accordproject/techdocs/wiki/Releases) and rather fork that information and update it with what this PR changes? We'll want to have something definitive for maintainers to reference when both creating a release note and also labeling PRs.